### ISSUE TITLE
API to get width and height of Reveal object

### ISSUE THREAD

**Issue by earboxer** [hakimel/reveal.js#2409]
There is a a `getScale()` method, there should also be

`getWidth()` and `getHeight()`,

which would be similar to `parseFloat(jQuery('.slides').css('height'))`

**Comment by earboxer** [comment]
Note: `jQuery('.slides').css('height')` doesn't behave the same in all jQuery versions.

Example case: default settings, your window is any size (within the scaling limits).

```js
Reveal.getWidth() === 960
Reveal.getHeight() === 700
```

Case: width: "100%", height: "100%", your window is 1024x768
```js
Reveal.getWidth() === 1024
Reveal.getHeight() === 768
```

Case: width: 960, height: "100%", your window is 1024x768 (after #2408 is merged in)
```js
Reveal.getWidth() === 960
Reveal.getHeight() === 720
```

**Comment by hakimel** [comment]
Merged in #2574
